### ISSUE TITLE
Error while running the compiler args parser script

### ISSUE THREAD

**Issue by DawnLightt** [microsoft/vscode-makefile-tools#385]
Hello, I'm having trouble with parsing dryrun.log. 
I have set logging level to debug a this is the output of log file: 
[log.txt](https://github.com/microsoft/vscode-makefile-tools/files/10011731/log.txt)
The errors just repeat for the rest of the project (I included only small portion of log file).

**Comment by bobbrow** [comment]
Thank you for reporting this issue.  @andreeis can you investigate this when you return?

**Comment by andreeis** [comment]
@DawnLightt, is "/tmp/parseCompilerArgs.sh" present on disk?

**Comment by andreeis** [comment]
@DawnLightt , does this still happen for you? We got another report with good hints https://github.com/microsoft/vscode-makefile-tools/issues/516. Hopefully your case is a dupe of that and our fix will unblock you as well. But in case the two issues are different, please confirm if you still see the symptom and give us the requested info.

**Comment by gcampbell-msft** [comment]
Closing as a duplicate of #516. Thanks!
